Userland/Applications/PixelPaint/MainWidget.cpp

```diff
@@ -577,6 +577,29 @@ void MainWidget::initialize_menubar(GUI::Window& window)
             editor->image().crop(content_bounding_rect.value());
             editor->did_complete_action("Crop Image to Content"sv);
         }));
+    m_image_menu->add_separator();
+    m_image_menu->add_action(GUI::Action::create(
+        "&Fit Canvas to Layers", [&](auto&) {
+            auto* editor = current_image_editor();
+            VERIFY(editor);
+            auto bounding_rect = editor->image().bounding_rect();
+            if (!bounding_rect.has_value())
+                return;
+            editor->image().set_canvas_size(bounding_rect.value());
+            editor->did_complete_action("Fit Canvas to Layers"sv);
+        }));
+    m_image_menu->add_action(GUI::Action::create(
+        "&Fit Canvas to Selection", [&](auto&) {
```

Comment:
You can't have the same `&` letter for two actions

Reply:
It is now possible to [cycle through menu items with the same Alt shortcut](https://github.com/SerenityOS/serenity/commit/288d75fdabf161cc47d7326d63334702969673ae).